### ISSUE TITLE
New Rule:  Inactive elements must not receive keyboard focus

### ISSUE THREAD

**Issue by A11YAngel** [opena11y/oaa-rules#28]
**Current Category**: Keyboard Support
**New Rule**:  Inactive elements must not receive keyboard focus
**Requirement or recommendation**:  Required
I overlooked the rule in Section 508, § 508-1194.21 (a) where it says that it should be executable or functional from the keyboard.

**Purpose/description**: 
Inactive elements that do not trigger, function or operate must not receive keyboard focus because keyboard-only users may be confused trying to activate the element and finding that the element does not function at all.

**Standards**:
§ 508-1194.21
(a) Executable and operable functions from a keyboard 
WCAG 2.0 Level A
2.1.1 Keyboard Accessibility
